bpf/bpf.go

```diff
@@ -91,14 +91,20 @@ var (
 	// v4Dot16Dot0 is the first kernel version that has all the
 	// required features we use for XDP filtering
 	v4Dot16Dot0 = versionparse.MustParseVersion("4.16.0")
-	// v4Dot20Dot0 is the first kernel version that has all the
-	// required features we use for sidecar acceleration
-	v4Dot20Dot0 = versionparse.MustParseVersion("4.20.0")
+	// v4Dot18Dot0 is the kernel version in RHEL that has all the
+	// required features for BPF dataplane, sidecar acceleration
+	v4Dot18Dot0 = versionparse.MustParseVersion("4.18.0-193")
```

Comment:
Unfortunately, I think the hashicorp version library we're using interprets  `x.y.z-1234` as coming before  `x.y.z`, which is not what we want.   (That's because it uses semver, which is not the scheme that redhat uses: https://play.golang.org/p/k_g9cOONGV5)